docs/conf.py

```diff
@@ -210,10 +212,33 @@
 """
 
 
+object_description_options_map["std:confval"] = dict(
+    toc_icon_class="data", toc_icon_text="C", generate_synopses="first_sentence"
+)
```

Comment:
Just an observation: It looks like this will only generate a synopsis when the `confval` role is used.

Reply:
The default value is "first_paragraph", so it will generate a synopsis by default, this just overrides it to use first_sentence.

Reply:
I understands the code here. I'm just noting that
```rst
.. confval:: html_theme_options

   Some synopsis statement.

   - This gets no synopsis: `html_theme_options`, but the xref target it resolved.
   - This gets a synopsis: :confval:`html_theme_options`.
```

Reply:
Ah, sorry --- I misunderstood.  Yes, I had forgotten to also monkey patch `resolve_any_xref`.  Now fixed.